api/alembic/versions/56916d46d8cb_high_hfi_area.py

```diff
@@ -0,0 +1,53 @@
+"""high_hfi_area
+
+Revision ID: 56916d46d8cb
+Revises: 7d240064d71a
+Create Date: 2022-12-08 15:28:34.992996
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '56916d46d8cb'
+down_revision = '7d240064d71a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('run_parameters',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('run_type', postgresql.ENUM('actual', 'forecast',
+                              create_type=False, name='runtypeenum'), nullable=False),
+                    sa.Column('run_datetime', sa.TIMESTAMP(timezone=True), nullable=False),
+                    sa.Column('for_date', sa.Date(), nullable=False),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('run_type', 'run_datetime', 'for_date'),
+                    comment='A combination of run type, run datetime and for date.'
+                    )
+    op.create_index(op.f('ix_run_parameters_id'), 'run_parameters', ['id'], unique=False)
+    op.create_table('high_hfi_area',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('advisory_shape_id', sa.Integer(), nullable=False),
+                    sa.Column('run_parameters', sa.Integer(), nullable=False),
+                    sa.Column('advisory_area', sa.Float(), nullable=False),
+                    sa.Column('warn_area', sa.Float(), nullable=False),
```

Comment:
I thiiiiiiiiink it's possible to add comments to Columns in sqlalchemy (?) If it is, I'd strongly recommend adding comments to describe the difference between an advisory and a warning

Reply:
@andrea-williams  I couldn't figure out how to annotate a column, but I updated the 'comment' for the table to be more descriptive. Would you prefer to see more descriptive column names perhaps?

Reply:
my bad, looks like SQLAlchemy lets us add comments on tables, constraints, indexes, basically everything EXCEPT for columns. 🙄

Reply:
I think having a SQL comment on the table is good enough. But I'm thinking it would probably help future devs if we explicitly state that advisory means HFI of 4-10,000, and warning means 10k+. And probably the models/auto_spatial_advisory.py is a better place to insert that comment in the code

Reply:
Agreed, I've added the same comment to the model.